code/datums/wires/airlock.dm

```diff
@@ -18,7 +19,7 @@
 
 /datum/wires/airlock/interactable(mob/user)
 	var/obj/machinery/door/airlock/A = holder
-	if(!issilicon(user) && A.isElectrified() && A.shock(user, 100))
+	if(!issilicon(user) && (SEND_SIGNAL(A, COMSIG_AIRLOCK_WIRES_INTERACT, user) & COMSIG_ELECTRIFIED_SHOCK))

```

Comment:
you should not have side effects in condition check functions like this

Reply:
I'll make a different PR to fix this, how should it be done?

Reply:
Disregard